### ISSUE TITLE
Non-test methods should not be converted to tests

### ISSUE THREAD

**Issue by sbleon** [jaredbeck/minitest_to_rspec#12]
In:
```
class PasswordsControllerTest < ActionController::TestCase
  def dumb_calculator
    2 + 2
  end
end
```

Out (expected):
```
RSpec.describe(PasswordsController, :type => :controller) do
  def dumb_calculator
    2 + 2
  end
end
```

Out (actual):
```
  it("dumb calculator") do
    2 + 2
  end
```

So i guess only methods starting with `test_`, and `it`, `should`, and `test` calls, should be converted to `it` specs.

**Comment by jaredbeck** [comment]
Fixed by #13, released in 0.10.2

**Comment by sbleon** [comment]
Thanks!
